src/wallet/wallet.cpp

```diff
@@ -3535,6 +3537,7 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
 
     wtxNew.fTimeReceivedIsTxTime = true;
     wtxNew.BindWallet(this);
+    LOCK(cs_main);
```

Comment:
This implies that the `LOCK2(cs_main, cs_wallet);` at line 3581 is redundant and should be changed to
```
// cs_main already taken above
LOCK(cs_wallet);
```

Reply:
Darn, I forgot to include the comment in the force-push I just did; do you think that's worth adding?